android/brave_android_wireguard_sources.gni

```diff
@@ -0,0 +1,25 @@
+import("//build/config/android/abi.gni")
+
+_brave_public_apk_or_module_tmpl_deps =
+    [ "//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar" ]
+
+brave_public_apk_or_module_tmpl_deps = _brave_public_apk_or_module_tmpl_deps
+brave_monochrome_public_apk_or_module_tmpl_deps =
+    _brave_public_apk_or_module_tmpl_deps
+
+_lib_dir = get_label_info(
+               "//brave/third_party/android_deps:com_wireguard_android($default_toolchain)",
+               "target_out_dir") + "/com_wireguard_android_java/jni"
+
+brave_public_apk_or_module_tmpl_loadable_modules =
+    [ "$_lib_dir/$android_app_abi/libwg-go.so" ]
+
+brave_monochrome_public_apk_or_module_tmpl_loadable_modules = []
+brave_monochrome_public_apk_or_module_tmpl_secondary_abi_loadable_modules = []
```

Comment:
this file should declare only wireguard-specific vars which then should be used in `brave_java_sources.gni`.
The issue is that if we want to add something else into `brave_monochrome_public_apk_or_module_tmpl_loadable_modules`, currently we should add this into "wireguard" file. This doesn't look right.

If you have some follow-ups, please fix this. 

1. All of this should be declared in `brave_android_wireguard_sources.gni`:
```
brave_wireguard_apk_deps = 
    [ "//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar" ]
brave_wireguard_apk_loadable_modules = 
    [ "$_lib_dir/$android_app_abi/libwg-go.so" ]
brave_wireguard_apk_secondary_abi_loadable_modules =
    [ "$_lib_dir/$android_app_secondary_abi/libwg-go.so" ]
```
2. All of this should be declared in `brave_java_sources.gni`:
```
brave_public_apk_or_module_tmpl_deps = []
brave_monochrome_public_apk_or_module_tmpl_deps = []

brave_public_apk_or_module_tmpl_loadable_modules = []
brave_monochrome_public_apk_or_module_tmpl_loadable_modules = []
brave_monochrome_public_apk_or_module_tmpl_secondary_abi_loadable_modules = []
```
and then you should write something like this:
```
brave_public_apk_or_module_tmpl_deps += brave_wireguard_apk_deps
brave_monochrome_public_apk_or_module_tmpl_deps += brave_wireguard_apk_deps

brave_public_apk_or_module_tmpl_loadable_modules += brave_wireguard_apk_loadable_modules
brave_monochrome_public_apk_or_module_tmpl_loadable_modules += brave_wireguard_apk_loadable_modules
brave_monochrome_public_apk_or_module_tmpl_secondary_abi_loadable_modules += brave_wireguard_apk_secondary_abi_loadable_modules
```